### PR TITLE
Added Quote Generator

### LINKED ISSUE #516: Add Quote generator
**Project title**
Quote generator

**Project Description**
The quote generator generates automatic quotes and colorful background after hitting the new quote button

**Stack**:  
And in order to tick the check box just put x inside them for example - [x] like this. Please delete options that are not relevant.

- [x] Html
- [x] CSS
- [x] JavaScript

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**
<hr>

## Description 📜

It was a simple Quote Generator app that generate new quotes after hitting the button

Fixes #516 

<hr>

## Type of change 📝

Please delete options that are not relevant. And in order to tick the check box just put x inside them for example [x] like this

- [x] Add a new project

<hr>

## Domain of Contribution 📊

Please delete options that are not relevant. And in order to tick the check box just put x inside them for example [x] like this

- [x] Project using HTML, CSS & JS

<hr>
 
## Checklist ✅

Please delete options that are not relevant. And in order to tick the check box just put x inside them for example [x] like this

- [x] I have performed a self-review of my own code or work.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I'm GSSoC'22 contributor

<hr>
 
## Screenshots / Gif (Optional) 📸

Screenshot (829)

Screenshot (830)

### CODE REVIEW

File: Quote Generator/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+It was a simple Quote Generator app that generate new quotes after hitting button
+
+## Tech Stacks
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![JS](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+
+## Screenshots
+
```
Comment (meinhoonharsh) [inline Quote Generator/README.md:11]: Screenshots Heading is there, but no Screenshot there 😓

**Review by meinhoonharsh — CHANGES_REQUESTED** [review]
All looks good, just that README thing

File: Quote Generator/README.md (hunk 1 above)
Comment (gurjeetsinghvirdee) [reply Quote Generator/README.md:11]: hehe I forgot to add😅

**Comment by khushi-purwar** [comment]
This project cannot be added as it will be treated as duplicate project. Quote generator is already present in this repo.

**Comment by khushi-purwar** [comment]
I am closing this PR.

**Pull request closed without merging**